robot/Cumulus/resources/locators_49.py

```diff
@@ -238,9 +238,12 @@
     'object_manager':{
         'button':'//input[@title="{}"]',
         'input':'//input[@id="{}"]',
-        'select_related':'//select[@id = "DomainEnumOrId"]',
+        'select_related':'//select[@id = "{}"]',
         'select_related_option':'//select[@id = "DomainEnumOrId"]/option[@value="{}"]',
         'search_result': '//tbody/tr/td/a/span[contains(text(),"{}")]',
+        'formula_txtarea': '//textarea[@id = "{}"]',
+        'input_validation': '//input[@title = "{}"]',
```

Comment:
line 239 and 245 have same locator, please remove one

Reply:
done